src/replication/replicator.rst

```diff
@@ -696,3 +728,71 @@ The syntax for a selector is the same as the
 Using a selector is significantly more efficient than using a JavaScript
 filter function, and is the recommended option if filtering on document
 attributes only.
+
+Specifying Usernames and Passwords
+===================================
+
+There are multiple ways to specify usernames and passwords for replication endpoints:
+
+ - In an ``{"auth": {"basic": ...}}`` object:
+
+    .. versionadded:: 3.2.0
+
+    .. code-block:: javascript
+
+        {
+            "target": {
+                "url": "http://someurl.com/mydb",
+                "auth": {
+                    "basic": {
+                        "username": "$username",
+                        "password": "$password"
+                     }
+                }
+            },
+            ...
+        }
+
+    This is the prefererred format as it allows including characters like ``@``, ``:``
+    and others in the username and password fields.
+
+ - In the userinfo part of the endpoint URL. This allows for a more compact
```

Comment:
I wonder if it's worth mentioning that URLs of this format are obsolete? Also, should we deprecate this format when using auth?

Reply:
Good idea

Reply:
I mentioned at the end of the section that it is deprecated in the RFC, but also mentioned we don't have a target removal version in CouchDB.